server/controllers/index.js

```diff
@@ -1,24 +1,62 @@
-const model = require('../models');
+const model = require("../models");
 
 module.exports = {
-
-  getListings: (req, res) => {
-    res.end();
+  getProducts: (req, res) => {
```

Comment:
I think we'll need both a "getListings" function and a "getProducts" function.

Reply:
getListings for populating the "marketplace". and getProducts for populating the listing page/create listing form.

Reply:
I've talked with Holly and Aaron, and I think we're good with using get Products to populate marketplace